packages/node_modules/@webex/plugin-meetings/src/media/index.js

```diff
@@ -2,10 +2,10 @@
  * Copyright (c) 2015-2018 Cisco Systems, Inc. See LICENSE file.
  */
 /* globals navigator */
-import {
-  AUDIOINPUT,
-  VIDEOINPUT
-} from '../constants';
+
+import Promise from 'bluebird';
```

Comment:
yuck but ok

Reply:
I could probably remove bluebird and do a different implementation of promise.try if you are passionate about it

Reply:
we should remove all bluebird at some point

Reply:
blasphemy!